### PR TITLE
Fix typos across yarn website repo

### PR DESCRIPTION
**Pull request by yohix**
:writing_hand: **Fix typos across yarn website repo**

**Files that have typos**

```
website/_posts/2017-07-11-lets-dev-a-package-manager.md:555:17: "becames" is a misspelling of "becomes"
website/lang/en/docs/yarnrc.md:59:43: "parallely" is a misspelling of "parallelly"
website/lang/en/package.html:7:50: "percieve" is a misspelling of "perceive"
```

**PS:** I use [**client9/misspell**](https://github.com/client9/misspell) to scan typos :heart: 

Making the world a better place :innocent:

### CODE REVIEW

File: lang/en/docs/yarnrc.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Value must be a boolean, defaults to `false`.
 child-concurrency #number#
 ```
 
-Controls the number of child processes run parallely to build node modules.
+Controls the number of child processes run parallelly to build node modules.
```
Comment (Haroenv) [inline lang/en/docs/yarnrc.md]:
```suggestion
Controls the number of child processes run in parallel to build node modules.
```

?

Comment (yohix) [reply lang/en/docs/yarnrc.md]: Controls the number of child processes **that** run in parallel to build node modules.

WDYT @Haroenv?

Comment (Haroenv) [reply lang/en/docs/yarnrc.md]: yes, that seems best, thanks!

**Comment by Haroenv** [comment]
thanks

**Pull request merged**